packages/postgres-database/src/simcore_postgres_database/utils_users.py

```diff
@@ -18,17 +21,17 @@ class UserNotFoundInRepoError(BaseUserRepoError):
 
 class UsersRepo:
     @staticmethod
-    async def get_role(conn: SAConnection, user_id: int) -> UserRole:
+    async def get_role(conn: SAConnection, user_id: int) -> str:
```

Comment:
Still don't agree here.
UserRole is the type defined in https://github.com/ITISFoundation/osparc-simcore/blob/2e5d96c42b89f6edd957f47db8c016a198719d56/packages/postgres-database/src/simcore_postgres_database/models/users.py

Why do you cast that again to a str?